### PR TITLE
Further decouple logical parts of frontend into separate Gradle modules

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java [hunk 1]
```diff
@@ -49,7 +50,8 @@ public void update(AnActionEvent anActionEvent) {
     super.update(anActionEvent);
 
     var presentation = anActionEvent.getPresentation();
-    var isReady = anActionEvent.getData(DataKeys.KEY_IS_GIT_MACHETE_REPOSITORY_READY);
+    var isReady = anActionEvent
+        .getData(com.virtuslab.gitmachete.frontend.keys.DataKeys.KEY_IS_GIT_MACHETE_REPOSITORY_READY);
```
Comment (mkondratek) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java:54]: Why not static import?

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/CheckOutBranchAction.java [hunk 2]
```diff
@@ -34,7 +36,8 @@ public void update(AnActionEvent anActionEvent) {
 
   @Override
   public void actionPerformed(AnActionEvent anActionEvent) {
-    String selectedBranchName = anActionEvent.getData(DataKeys.KEY_SELECTED_BRANCH_NAME);
+    String selectedBranchName = anActionEvent
+        .getData(com.virtuslab.gitmachete.frontend.keys.DataKeys.KEY_SELECTED_BRANCH_NAME);
```
Comment (mkondratek) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/CheckOutBranchAction.java:40]: Why not static import?

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java (hunk 1 above)
Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java]: Done, fixed

File: frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTableManager.java [hunk 3]
```diff
@@ -97,11 +99,9 @@ private void refreshGraphTable(Path macheteFilePath, boolean isMacheteFilePresen
     GuiUtils.invokeLaterIfNeeded(gitMacheteGraphTable::updateUI, ModalityState.NON_MODAL);
   }
 
-  /**
-   * Lambda inside ths function is invoked by {@link VcsRootDropdown#setValue()} when user changes repository in dropdown menu
-   */
   private void subscribeToVcsRootChanges() {
-    vcsRootDropdown.subscribe(this::updateAndRefreshInBackground);
+    // The lambda is invoked when user changes repository in dropdown menu
```
Comment (micpiotrowski) [inline frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTableManager.java:103]:
```suggestion
    // The method reference is invoked when user changes repository in dropdown menu
```

**Pull request merged**